src/main/java/com/google/devtools/build/lib/rules/cpp/LinkBuildVariables.java

```diff
@@ -158,24 +159,24 @@ public static CcToolchainVariables.Builder setupLtoIndexingVariables(
   }
 
   public static CcToolchainVariables.Builder setupLinkingVariables(
-      String outputFile,
+      Artifact outputFile,
```

Comment:
Does anything need to change in src/main/starlark/builtins_bzl/common/cc/link/link_build_variables.bzl?

I'm planning to flip on Starlark linking code this week. I don't want to undo this changes or redo them.

Reply:
I updated `link_build_variables.bzl` with the corresponding changes. I will check whether the new test also passes with the Starlark implementation.

Reply:
Thanks!